apps/dashboard/src/components/workflow-editor/steps/email/maily-config.tsx

```diff
@@ -55,5 +61,20 @@ export const getDefaultEditorBlocks = (isCustomEmailBlocksEnabled: boolean): Blo
     ],
   });
 
+  blocks.push({
+    title: 'Components',
+    commands: [
+      {
+        id: 'headers',
+        title: 'Headers',
+        description: 'Header: Logo with cover image',
+        searchTerms: ['header', 'headers'],
+        icon: <EmailHeader className="size-4" />,
+        preview: '/images/email-editor/header-preview.png',
+        commands: [headerLogoWithCoverImage, headerCenteredLogoWithBorder, headerLogoWithText],
+      },
```

Comment:
I think the "subcommands" needs to have preview as well based on the Figma designs

Reply:
done